src/components/v5/frame/NavigationSidebar/NavigationSidebar.tsx

```diff
@@ -128,11 +128,14 @@ const NavigationSidebarContent: FC<NavigationSidebarProps> = ({
                   toggleOffThirdLevelMenu();
                 }}
                 className={clsx(
-                  'rounded-full border-[.1875rem] transition-all',
+                  'rounded-full border-[.1875rem] transition-all sm:hover:border-blue-400',
                   {
-                    'border-blue-400': openItemIndex === 0 && isTablet,
+                    'border-blue-400':
+                      (openItemIndex === 0 && isTablet) ||
+                      (openItemIndex === 0 && isSecondLevelMenuOpen),
                     'border-transparent':
-                      !isTablet || (openItemIndex !== 0 && isTablet),
+                      (!isTablet && !isSecondLevelMenuOpen) ||
```

Comment:
I don't understand these changes. I guess that you just want to set it as transparent when it's not expanded, then you should just check if openItemIndex is not 0